packages/web3torrent/tracker/index.js

```diff
@@ -11,6 +11,7 @@ const server = new Server({
   http: true,
   ws: true,
   stats: true,
+  interval: 150000,
```

Comment:
this does all the magic. 
Ignore the actual number. that 150000 is actually 30 seconds.
I don't understand the logic about how is the actual interval calculated.